src/lib/components/layout/Sidebar/SearchModal.svelte

```
@@ -0,0 +1,183 @@
+		archived?: boolean;
+		tags?: string[];
+	}
+
+	const i18n = getContext('i18n');
+
+	export let show = false;
+
+	let search = '';
+	let searchResults: Chat[] = [];
+	let loading = false;
+	let searchDebounceTimeout: NodeJS.Timeout | null = null;
```

ESLint

[error] 'NodeJS' is not defined.

(no-undef)